k8-util/helm/fluvio-sys/templates/crd_managed_connector.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: managedconnectors.fluvio.infinyon.com
+spec:
+  group: fluvio.infinyon.com
+  scope: Namespaced
+  names:
+    kind: ManagedConnector
+    plural: managedconnectors
+    singular: managedconnector
+  versions:
+    - name: v1
+      served: true
+      storage:  true
+      subresources:
+          status: {}
+      schema:
+        openAPIV3Schema:
+          required: ["spec"]
+          type: object
+          properties:
+            status:
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
+            spec:
+              type: object
+              required: ["name", "config"]
+              properties:
+                name:
+                  type: string
+                  minimum: 2
+                  maximum: 100
+                config:
+                  type: object
+                  required: ["type", "topic"]
+                  properties:
+                    type:
+                      type: string
+                      minimum: 2
+                      maximum: 100
+                    topic:
+                      type: string
+                      minimum: 2
+                      maximum: 100
+                    args:
+                      type: array
+                      items:
+                        type: string
```

Comment:
shouldn't be be better as key=value? that way it's better to enforce checking

Reply:
args are specific to each connector package, we cannot know or enforce the types here